lib/createlocaltracks.js

```diff
@@ -102,14 +102,15 @@ function createLocalTracks(options) {
       : {}
   };
 
-  // eslint-disable-next-line no-undefined
-  if (options.audio && options.audio.workaroundWebKitBug1208516 !== undefined) {
-    localTrackNameOptions.audio.workaroundWebKitBug1208516 = options.audio.workaroundWebKitBug1208516;
+  extraLocalTrackOptions.audio.isCreatedByCreateLocalTracks = true;
```

Comment:
since this is always assigned I would move it inside the declaration of `extraLocalTrackOptions` like
```
const extraLocalTrackOptions = {
   audio: { isCreatedByCreateLocalTracks: true },
  video: { isCreatedByCreateLocalTracks: true}
}
```

and then add `name` and `workaroundWebKitBug1208516` conditionally here.

Reply:
@makarandp0 I think this is OK for now.